src/platform/Darwin/CHIPPlatformConfig.h

```diff
@@ -53,9 +55,16 @@ extern "C" int __cxa_atexit(void (*f)(void *), void * p, void * d);
 #define CHIP_CONFIG_MAX_UNSOLICITED_MESSAGE_HANDLERS 8
 #endif // CHIP_CONFIG_MAX_UNSOLICITED_MESSAGE_HANDLERS
 
-#ifndef CHIP_CONFIG_MAX_EXCHANGE_CONTEXTS
-#define CHIP_CONFIG_MAX_EXCHANGE_CONTEXTS 8
-#endif // CHIP_CONFIG_MAX_EXCHANGE_CONTEXTS
+//
+// Default of 8 ECs is not sufficient for some of the unit tests
+// that try to validate multiple simultaneous interactions.
```

Comment:
This should only matter when building those unit tests, right?

That said, in general we might in fact want more than 8 on darwin.....

Reply:
It looks like this macro is being used to size an array as well.

from `src/lib/core/CHIPConfig.h`
```
#define CHIP_CONFIG_MCSP_RECEIVE_TABLE_SIZE (CHIP_CONFIG_MAX_EXCHANGE_CONTEXTS - 2)
```

from `src/protocols/secure_channel/MessageCounterManager.h`:
```
ReceiveTableEntry mReceiveTable[CHIP_CONFIG_MCSP_RECEIVE_TABLE_SIZE];
```